spec/features/visit_home_pages_spec.rb

```diff
@@ -16,6 +16,6 @@
     tag_component = PiwikTagComponent.new
     expect(tag_component).to be_in_html
     expect(tag_component.tracker_url_assigment).to include('piwik.example.com')
-    expect(tag_component.to_s).to eq fixture('index_example_com.html')
+    expect(tag_component.as_ecma).to eq fixture('index_example_com.js')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.